### PR TITLE
Remove graphik font

### LINKED ISSUE #153: source contains non-free font family Graphik
Source contains multiple instances of the font family Graphik, which is copyright https://commercialtype.com/ and licensed only under a non-free license.

### PR DESCRIPTION
**Pull request by daleglass**
Fixes #153

### CODE REVIEW

**Comment by vegaslon** [comment]
Think most of this changes will go unnoticed, The only real difference will want to check out is how the jsdocs look after this change.

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -1096,9 +1096,9 @@ Application::Application(
     QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Raleway-Bold.ttf");
     QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Raleway-SemiBold.ttf");
     QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Cairo-SemiBold.ttf");
-    QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Graphik-SemiBold.ttf");
-    QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Graphik-Regular.ttf");
-    QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Graphik-Medium.ttf");
+    QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Raleway-SemiBold.ttf");
```
Comment (ksuprynowicz) [inline interface/src/Application.cpp:1099]: This line occurs twice

File: interface/src/Application.cpp [hunk 2]
```diff
@@ -1096,9 +1096,9 @@ Application::Application(
...
+    QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Raleway-Regular.ttf");
+    QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Raleway-Medium.ttf");
```
Comment (ksuprynowicz) [inline interface/src/Application.cpp:1101]: fonts/Raleway-Medium.ttf file doesn't exist

**Comment by daleglass** [comment]
Notes for testing:

RadioButton.qml contained:
`topPadding: -3 // For perfect alignment when using Graphik`

and Button.qml contained:
`topPadding: -2 // Necessary for proper alignment using Graphik Medium`

So buttons and radio buttons should be looked at to see whether some sort of padding is needed and how much. I commented the line out entirely, so alignment might be wrong with the PR as-is.

File: launchers/win32/Launcher.rc [hunk 3]
```diff
@@ -77,9 +77,9 @@ IDR_MAINFRAME           ICON                    "res\\interface.ico"
 // BINARY
 //
 
-IDR_FONT_REGULAR        BINARY                  "res\\Graphikregular.ttf"
+IDR_FONT_REGULAR        BINARY                  "res\\Ralewayregular.ttf"
```
Comment (ksuprynowicz) [inline launchers/win32/Launcher.rc:80]: This file doesn't exist

File: launchers/win32/Launcher.rc [hunk 4]
```diff
@@ -77,9 +77,9 @@ IDR_MAINFRAME           ICON                    "res\\interface.ico"
...
 
-IDR_FONT_BOLD           BINARY                  "res\\Graphikbold.ttf"
+IDR_FONT_BOLD           BINARY                  "res\\Ralewaybold.ttf"
```
Comment (ksuprynowicz) [inline launchers/win32/Launcher.rc:82]: This file doesn't exist

**Review by ksuprynowicz — CHANGES_REQUESTED** [review]

File: launchers/win32/Launcher.rc (hunk 3 above)
Comment (daleglass) [reply launchers/win32/Launcher.rc:80]: We're not using the launcher anyway, so this seems optional

Comment (ksuprynowicz) [reply launchers/win32/Launcher.rc:80]: Agreed.

File: interface/src/Application.cpp (hunk 2 above)
Comment (daleglass) [reply interface/src/Application.cpp:1101]: File added

File: interface/src/Application.cpp (hunk 1 above)
Comment (daleglass) [reply interface/src/Application.cpp:1097]: fixed

**Comment by ksuprynowicz** [comment]
It's ready for testing :)

**Comment by ksuprynowicz** [comment]
I tested it on Linux and didn't notice any problems.

**Review by JulianGro — CHANGES_REQUESTED** [review]
A lot of the same issues as https://github.com/overte-org/overte/pull/155

**Comment by AnimatedMannequin** [comment]
Here are some builds you may use for testing: 

**ubuntu-18.04, full**
 - https://public.overte.org/build/overte/PR154/Overte-PR154-873399a.tar.gz
 - https://public.overte.org/build/overte/PR154/Overte-PR154-873399a.sh
 - https://public.overte.org/build/overte/PR154/Overte-PR154-873399a.tar.Z

**macOS-10.15, full**
 - https://public.overte.org/build/overte/PR154/Overte-PR154-873399a.dmg

**windows-2019, full**
 - https://public.overte.org/build/overte/PR154/Overte-PR154-873399a.exe

**self-hosted_debian-11_aarch64, full**
 - https://public.overte.org/build/overte/PR154/Overte-PR154-873399a.sh
 - https://public.overte.org/build/overte/PR154/Overte-PR154-873399a.tar.Z
 - https://public.overte.org/build/overte/PR154/Overte-PR154-873399a.tar.gz

**Pull request closed without merging**